src/components/ProgramRecord/ProgramRecord.jsx

```diff
@@ -53,6 +54,12 @@ function ProgramRecord({ isPublic }) {
   }, [programId, isPublic]);
 
   const toggleSendRecordModal = () => {
+    if (!sendRecord.sendRecordModalOpen) {
+      sendTrackEvent('edx.bi.credentials.program_record.send_started', {
+        category: 'records',
+        'program-uuid': programId,
```

Comment:
Why is the key `'program-uuid'` surrounded in quotes but `category` isn't? Is this just to match current convention?

Reply:
This is the way is was written in the legacy frontend. However, that frontend uses an older Analytics helper function (inside its own `Analytics.js` file). The MFEs seem to have adopted a different method by importing a function from `frontend-platform/analytics`. As far as I can tell, these are just two methods for accomplishing the same thing so the arguments for each tracking function should be almost identical. I don't think these quotes are necessary, however. The older MFE's don't use quotes on any keys so I'm thinking I should get rid of them.